### ISSUE TITLE
[Issue] - Health Checker reports missing web.config but information is outdated

### ISSUE THREAD

**Issue by kenmich** [microsoft/CSS-Exchange#1341]
**Provide Version Number**
22.11.09.1711

**Describe the issue**
Running Health Checker on multiple Exchange 2019 (CU12 Nov22SU) servers, reports the following:

```
Missing Configuration File:
        Missing: C:\inetpub\AutodiscoverRoot\web.config
        More Information: https://aka.ms/HC-MissingConfig
```

**Expected behavior**
The article referenced in "More Information" relates to an article that seems unrelated and irrelevant.

**Additional context**
We see no issues with autodiscover, so the missing configuration file might not be a real issue or if so, the provided article is irrelevant for the specific issue.

**Comment by dpaulson45** [comment]
For some reason you have a virtual directory that would have the configuration files located at `c:\inetpub\AutodiscoverRoot\` which is not typical as it should be located inside of the Exchange Install directory. There is something likely additional on your server that we just don't know how to handle because this isn't typical.

Do you have a 3rd site in IIS? Are you able to provide the output from your Health Checker with `-Verbose` to include the debug log?

**Comment by lusassl-msft** [comment]
@kenmich If you can/want to share the verbose output (`.\HealthChecker.ps1 -SaveDebugLog`) with us, feel free to email the log to: `ExToolsFeedback[at]microsoft[dot]com`

**Comment by kenmich** [comment]
Yes, we are running two extra websites in IIS, in order to handle autodiscover redirect. It was done after a Microsoft best practice, although I am unable to find the article at the moment. Maybe it was lost when moving to Microsoft Learn.

It's not a production issue, but just the warning that is irrelevant. I have submitted the debug log via. e-mail as asked.

**Comment by lusassl-msft** [comment]
Is it this article? 
https://techcommunity.microsoft.com/t5/exchange-team-blog/configuring-multiple-owa-ecp-virtual-directories-on-the-exchange/ba-p/611217

It states:
_Microsoft supports using multiple Outlook Web App (OWA) and Exchange Control Panel/Admin Center (ECP) front end virtual directories on a server with the Exchange 2013 Client Access Server role, when each is in its own website and is named ‘OWA’ and ‘ECP’.  Each virtual directory must be listening on the standard TCP 443 port for the site._

So, if you have multiple AutoDiscover vDirs in place, this might be an unsupported scenario (have to investigate).

**Comment by kenmich** [comment]
No it was not that specific article. it was specifically for autodiscover redirect.

**Comment by lusassl-msft** [comment]
Confirmed - we received your email.
If I remember correctly, there was something like this described for Exchange Server 2007. If you find the article, please share it with us.

**Comment by kenmich** [comment]
There are no more warnings with version 22.12.13.1814. All looks good now!
